Core/src/org/sleuthkit/autopsy/corecomponents/DataResultViewerThumbnail.java

```diff
@@ -384,7 +384,9 @@ public void setNode(Node givenNode) {
         }
         try {
             if (givenNode != null) {
-                tfn = (TableFilterNode) givenNode;
+                if (givenNode instanceof TableFilterNode){
+                    tfn = (TableFilterNode) givenNode;
+                }
```

Comment:
I'm not sure how this relates to artifacts as content cleanup. All nodes that are pushed from the results view to the tree view are expected to be wrapped with TableFilterNode decorators, so a bad cast exception is probably a good thing to draw our attention to a programming error.  Did you encounter an error here?

Reply:
Sorry this change got accidentally included as it was part of me trouble shooting another bug before I wrote it up, regarding the thumbnail view in timeline mode throwing errors regarding being unable to cast to a TableFilterNode.